server/unit/ultrablue-server.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=ultrablue remote attestation service
+After=bluetooth.service
+Wants=cryptsetup-pre.target
+Before=cryptsetup-pre.target
+DefaultDependencies=no
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/ultrablue-server
+RemainAfterExit=yes
```

Comment:
Out of curiosity, why is this necessary? Is there another service that depends on this one staying active even after the server exits?

Reply:
Are you talking about the RemainAfterExit= option ? I like the idea to keep it active for user informative purpose, it can be dropped without functional drawbacks